ts/features/pn/utils/index.ts

```diff
@@ -115,3 +115,11 @@ export const containsF24FromPNMessagePot = (
     O.getOrElse<ReadonlyArray<ThirdPartyAttachment>>(() => []),
     RA.some(attachment => attachment.category === ATTACHMENT_CATEGORY.F24)
   );
+
+export const canShowMorePaymentsLink = (
+  isCancelled: boolean,
+  payments?: ReadonlyArray<NotificationPaymentInfo>
+): payments is ReadonlyArray<NotificationPaymentInfo> =>
+  !isCancelled &&
+  !!payments &&
+  payments.length > maxVisiblePaymentCountGenerator();
```

Comment:
```suggestion
  payments?.length > maxVisiblePaymentCountGenerator();
```
small "eye candy" change, the style is up to the dev after all, so feel free to reject this.

Reply:
`payments?.length > maxVisiblePaymentCountGenerator();`
produces the following error 
`TS18048: 'payments.length' is possibly 'undefined'`